testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakConfiguration.java

```diff
@@ -52,6 +52,10 @@ public class KeycloakConfiguration {
     public static final String CHARLIE = "charlie";
     public static final String CHARLIE_PASSWORD = "charlie123+";
     public static final String ALLOWED_ORIGIN = "http://somehost";
+    public static final String ALICE_First_Name = "Alice";
+    public static final String ALICE_LAST_Name = "Smith";
```

Comment:
For constants, variable names can be all upper case. The two variable names above are mixing lower case and upper case.